hyperspy_gui_traitsui/utils.py

```diff
@@ -3,10 +3,6 @@
 from hyperspy.ui_registry import register_widget
```

Comment:
This import needs to be remove, because `register_widget` does not exist anymore.